flow/main.go

```diff
@@ -207,11 +211,12 @@ func main() {
 					temporalHostPort := clicmd.String("temporal-host-port")
 
 					return cmd.APIMain(ctx, &cmd.APIServerParams{
-						Port:              uint16(clicmd.Uint("port")),
-						TemporalHostPort:  temporalHostPort,
-						GatewayPort:       uint16(clicmd.Uint("gateway-port")),
-						TemporalNamespace: clicmd.String("temporal-namespace"),
-						EnableOtelMetrics: clicmd.Bool(otelMetricsFlag.Name),
+						Port:                             uint16(clicmd.Uint("port")),
+						TemporalHostPort:                 temporalHostPort,
+						GatewayPort:                      uint16(clicmd.Uint("gateway-port")),
+						TemporalNamespace:                clicmd.String("temporal-namespace"),
+						EnableOtelMetrics:                clicmd.Bool(otelMetricsFlag.Name),
+						RunScheduledTasksWithoutTemporal: clicmd.Bool("run-scheduled-tasks-without-temporal"),
```

Comment:
Little messy having flag name repeated here, may clean this up a bit here after #3266 merges